complaints/templates/complaints/contact_before.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
```

Comment:
You already have contact.html.Then what is this file for?
Please remove such duplicate and unnecessary files.